src/edges_cal/modelling.py

```diff
@@ -260,6 +260,18 @@ def transform(self, x: np.ndarray) -> np.ndarray:
         return np.log(x)
 
 
+@h5.hickleable()
+@attr.s(frozen=True, kw_only=True)
+class Log10Transform(ModelTransform):
+    """A transform that takes the logarithm of the input."""
+
+    scale: float = attr.ib(1.0)
+
+    def transform(self, x: np.ndarray) -> np.ndarray:
+        """Transform the coordinates."""
+        return np.log10(x / self.scale)
```

Comment:
@steven-murray Sorry I added the logpoly model to this branch instead of creating  a new one. 
Also, there is an error in this one. And I don't know how to fix it with the way this code is set up. 

So we want 10**sum(terms) Not sum(10**term). The latter is how I have seemed to defined it as

Reply:
OK, I think I understand what you're saying. This code looks good, but I think what you want to do is not do 10**term, but instead take the log10 of the data you feed to fit.